File: src/main/java/org/pyjjs/scheduler/core/actors/common/locale/LangResolver.java
package org.pyjjs.scheduler.core.actors.common.locale;

public class LangResolver {

    private static final String LOCALE_FILE_NAME = "locale_%s.properties";
    private String fileName;
    private String shortLang;


    public LangResolver(String shortLang) {
        fileName = String.format(LOCALE_FILE_NAME, shortLang);
        this.shortLang = shortLang;
    }

    public String getFileName() {
        return fileName;
    }

    public String getShortLang() {
        return shortLang;
    }
}

File: src/main/kotlin/org/pyjjs/scheduler/core/common/locale/LangResolver.kt
package org.pyjjs.scheduler.core.common.locale

class LangResolver(val shortLang: String) {
    val fileName: String
    init {
        fileName = String.format(LOCALE_FILE_NAME, shortLang)
    }

    companion object {
        private val LOCALE_FILE_NAME = "locale_%s.properties"
    }
}
